src/assets/scripts/client/constants/gameOptionConstants.js

```diff
@@ -34,6 +35,16 @@ export const GAME_OPTION_VALUES = [
         ]
     },
     {
+        name: GAME_OPTION_NAMES.PTL_LENGTH,
+        defaultValue: '1',
+        description: 'Projected Track Line (PTL) Length',
+        type: 'select',
+        data: [
+            ['1', '1'],
+            ['0.5', '0.5']
```

Comment:
there may have been an operator error on that one, cause I can't recreate it anymore either.

easy options to add.  I may actually take the time to re-work these `gameOptions` a little bit because its a little clunky.